dev/com.ibm.ws.ejbcontainer.core/resources/com/ibm/ejs/container/container.nlsprops

```diff
@@ -1561,6 +1561,11 @@ NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W=CNTR0340W: The {2} Java Naming an
 NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W.explanation=The local binding name can only contain one namespace and it must start with ejblocal:. Namespaces are denoted by the colon (:) character. The entire string before and including the colon (:) character is considered a namespace.
 NAMESPACE_IN_LOCAL_JNDI_BINDING_NAME_CNTR0340W.useraction=Correct the ibm-ejb-jar-bnd.xml bindings file to give this bean a valid JNDI binding name by removing the namespace that is not valid.
 
+#-------------------------------------------------------------------------------
+UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W=CNTR0341W: Unable to bind the {0} interface of the {1} bean in the {2} module of the {3} application to the {4} name location. The ORB service is unavailable.
```

Comment:
what does ORB stand for? If it's not defined anywhere we should spell out on first mention.

Reply:
Ultimately, this is the message that is missing from the logs that results in this condition:
```
CWWKI0001I: The CORBA name server is now available at corbaloc:iiop:localhost:2809/NameService.
```
So, rather than specifically indicate ORB, should our message relate to that message, perhaps like this: :
```
... The CORBA name server is not available.
```